src/services/editor.service.ts

```diff
@@ -101,4 +102,50 @@ export class EditorService {
       throw err;
     }
   }
+
+  static async convertToYaml() {
```

Comment:
Why is it an `async` function? 🤔 I can see it is for the toaster, but there is no asynchronous code, is it really needed? 😄

Reply:
Yes, `async` is needed, because I wanna treat function as Promise to not block the main thread that renders the html - every state/actions in front should be asynchronous.

Reply:
Fair 👍